src/main/java/net/minecraftforge/common/world/ClimateSettingsBuilder.java

```diff
@@ -6,7 +6,6 @@
 package net.minecraftforge.common.world;
 
 import net.minecraft.world.level.biome.Biome.ClimateSettings;
-import net.minecraft.world.level.biome.Biome.Precipitation;
```

Comment:
@sciwhiz12 Why does this remove an import? Is this done on purpose or by accident?

Reply:
Yeah that is fine. Already checked :D